languages/r-3/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "artifacts": [
+        {"source":"/home/algo/.local", "destination":"/home/algo/.local/"},
```

Comment:
this directory doesn't seem to be used - it's the pip local install directory.

Reply:
Removed in b3bc0d2ba2d7dd09af61f12b1ba84e315b015fd0.